interface/src/state.rs

```diff
@@ -537,11 +543,13 @@ impl Delegation {
             })
         {
             // target_epoch > self.deactivation_epoch
+            //
+            // We advance epoch-by-epoch from just after the deactivation epoch up to the target_epoch,
+            // removing (cooling down) the account's share of effective stake each epoch,
+            // potentially rate-limited by cluster history.
```

Comment:
Code comments + variable name change below were to add clarity in areas I had some trouble mentally grasping